glance/server.sls

```diff
@@ -79,6 +79,27 @@ glance_install_database:
   - require:
     - service: glance_services
 
+{%- if server.get('image_cache', {}).get('enabled', False) %}
+glance_cron_glance-cache-pruner:
```

Comment:
Unfortunately I don't :-/ There could be some parameter to control whether cron jobs should be deployed (eventually override when they should get executed).